mapbox_tilesets/scripts/cli.py

```diff
@@ -703,3 +705,70 @@ def list_sources(username, token=None):
             click.echo(source["id"])
     else:
         raise errors.TilesetsError(r.text)
+
+
+@cli.command("estimate-area")
+@cligj.features_in_arg
+@click.option(
+    "--precision",
+    "-p",
+    required=True,
+    type=click.Choice(["10m", "1m", "30cm", "1cm"]),
+    help="Precision level",
+)
+@click.option(
+    "--no-validation",
+    required=False,
+    is_flag=True,
+    help="Bypass source file validation",
+)
+@click.option("--force-1cm", required=False, is_flag=True, help="Enables 1cm precision")
```

Comment:
the formatting on this option doesn't match the two above. 

could we update it to match OR do we have a linter that we use on this repo?